### ISSUE TITLE
add CITATION.cff file

### ISSUE THREAD

**Issue by bgruening** [federicomarini/ideal#12]
We talked about this one :)
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

**Comment by federicomarini** [comment]
Yep - Would be interesting to see if there is a bibtex2cff utility around.

While at it - should go in another issue, but I'll speak up: License files, should they be added explicitly as well?

**Comment by bgruening** [comment]
Yes to LICENSE files.

**Comment by federicomarini** [comment]
I am just wondering - since R packages have already the "standardized way" of making them citable with the item in `inst/citation`, shall this one come on top? I have the impression it is a bit redundant.
@csoneson, got any idea especially re: "regulations" in Bioc?
